chef/cookbooks/bind9/recipes/default.rb

```diff
@@ -293,9 +293,12 @@ def make_zone(zone)
   end
 end
 
-# We would like to bind service only to ip address from admin network
+# We would like to bind service only to ip address from admin network unless enable_designate is
+# enabled. In which case bind both the admin _and_ public.
 admin_network = Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "admin")
 admin_addr = admin_network.address
+public_addr = Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "public").address
```

Comment:
why do we put this here and not inside the if below (line 403ff) ? it is only used once. I find it pretty cluttery to handle this generically here if its only for the designate case..